sys/net/gnrc/link_layer/lwmac/lwmac.c

```diff
@@ -578,6 +611,13 @@ static void _tx_management(gnrc_netif_t *netif)
             /* TX packet will be dropped, no automatic resending here. */
             /* Intentionally falls through */
 
+            /* Re-initialize the radio when needed. */
+            if (netif->mac.tx.preamble_fail_counts >= CONFIG_GNRC_LWMAC_RADIO_REINIT_THRESHOLD) {
+                netif->mac.tx.preamble_fail_counts = 0;
+                LOG_INFO("[LWMAC] Re-initialize radio.");
+                lwmac_reinit_radio(netif);
+            }
```

Comment:
This removes the compile-time warning.

```suggestion

            /* Re-initialize the radio when needed. */
            if (netif->mac.tx.preamble_fail_counts >= CONFIG_GNRC_LWMAC_RADIO_REINIT_THRESHOLD) {
                netif->mac.tx.preamble_fail_counts = 0;
                LOG_INFO("[LWMAC] Re-initialize radio.");
                lwmac_reinit_radio(netif);
            }
            /* Intentionally falls through */
```